apps/server/src/modules/class/service/class.service.ts

```diff
@@ -1,11 +1,14 @@
 import { Injectable, InternalServerErrorException } from '@nestjs/common';
 import { EntityId } from '@shared/domain/types';
+import { LegacyLogger } from '@src/core/logger';
```

Comment:
Please don't use anywhere LegacyLogger, but

https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/core/logger/logger.ts

Reply:
i used logger, and made in shared/common/loggable data-deletion-domain-operation-laggable class to use in each module